### DISCUSSION TITLE
Github discussions not being created on releases

### DISCUSSION THREAD

**Discussion by eapolinario** [goreleaser/goreleaser#4902] [Q&A] [answered]
As described in the [docs](https://goreleaser.com/customization/release/#github), I have `discussion_category_name` set to `General` in my [`.goreleaser.yml`](https://github.com/flyteorg/flyte/blob/master/.goreleaser.yml#L10). This should produce a Github discussion on releases, but this stopped working for my project some time after 3/29.

The evidence I collected so far:
- goreleaser created the [Flyte 1.11.1-b1 release](https://github.com/flyteorg/flyte/releases/tag/v1.11.1-b1) in this [github job](https://github.com/flyteorg/flyte/actions/runs/8486209787/job/23252345054) and the corresponding [github discussion](https://github.com/flyteorg/flyte/discussions/5151) on 3/29. Notice that this is running goreleaser 1.24.0
- the next release, [Flyte 1.12.0-b0](https://github.com/flyteorg/flyte/releases/tag/v1.12.0-b0), happened on 4/29 and it ran [goreleaser 1.25.1](https://github.com/flyteorg/flyte/actions/runs/8886391471/job/24400643292#step:4:11), but I see no corresponding github discussion in https://github.com/flyteorg/flyte/discussions/categories/general.
- The subsequent releases do not have github discussions.

I don't see any error in the goreleaser logs. How can I debug this further?

**Comment by davidmirror-ops** [comment]
You'll find a [new Discussion](https://github.com/flyteorg/flyte/discussions/5442) created for a recent release. It was created manually to unblock the release process.

**Comment by caarlos0** [comment]
I think this might be a github bug... the thing related to releases that changed in v1.25 is that now release is always created as a draft, and then published after the artifacts are pushed to it.

Maybe github is not creating the discussion then? Not sure 🤔

**Reply by eapolinario** [reply]
@caarlos0 , thanks for getting back to me.


> Maybe github is not creating the discussion then?

If that's the case, this feature should be broken for everyone, right? Have you got any other complaints? I couldn't find any [issue](https://github.com/goreleaser/goreleaser/issues) on github.

Is there any debug flag that I can set to help debug this?

**Reply by caarlos0** [reply]
so far no complaints, no... that's what is bugging me as well... 

my plan is to setup a demo project to test this out... just haven't got into doing it yet

**Comment by caarlos0** [comment, marked as answer]
Fixed in afd92ffe0fc1070081dcd7e0e49cf87f0749853f
